chia/_tests/core/custom_types/test_proof_of_space.py

```diff
@@ -286,7 +286,7 @@ def test_v1_phase_out() -> None:
     constants = DEFAULT_CONSTANTS.replace(HARD_FORK2_HEIGHT=uint32(500000))
     rng = random.Random()
 
-    phase_out_epochs = 1 << constants.PLOT_V1_PHASE_OUT_EPOCH_BITS
+    phase_out_epochs = (1 << constants.PLOT_V1_PHASE_OUT_EPOCH_BITS) - 1
```

Comment:
we should have one place in the codebase to get this calculation from, probably just a function the receives the Bits